docs/other/guides/how-to-create-scaffolding-templates.md

```diff
@@ -0,0 +1,45 @@
+---
+id: creatingScaffoldingTemplates
+title: How to create Scaffolding Templates
+sidebar_label: Creating Scaffolding Templates
+---
+
+### Introduction
+As discussed in [Scaffolding](/docs/get-started/scaffolding), it is possible to save yourself a lot of time by using one of the default Scaffolding Templates.
+
+The great news are that you may also write your own templates! In this article we will go in depth about it.
+
+### Writing your first Scaffolding Template
+We use `node modules` for all of our templates. You should create a new package whose name follows the following pattern: `webiny-cli-scaffold-*`.
```

Comment:
`We use node modules for all of our templates` - took me a while to understand what you meant by this. I'd rephrase it to `Webiny CLI loads templates that are available as dependencies of your project. Under the hood, we use "yarn list" to find the available template packages.`

Reply:
yeah, they aren't ruby gems, python packages or whatever else...

but since we only use node i suppouse it's redundant to mention taht

Reply:
done